src/Components/Main/Pages/CreatingPage/BasicSettingsForm/RowForm/RowForm.tsx

```diff
@@ -0,0 +1,32 @@
+import { type ChangeEvent, type ReactElement, memo } from 'react';
+import classes from './RowForm.module.scss';
+import type { FormikErrorsType } from '../../../../../../utils/types/form';
+import { FormField } from '../../../../../Common/FormField/FormField';
+import { FormName } from '../../../../../../utils/types/enums';
+
+type PropsType = {
+  labelIndex: number;
+  setLabels: (id: number, value: string) => void;
+  removeRow: (index: number) => void;
+  errors: FormikErrorsType;
+};
+
+export const RowForm = memo<PropsType>(({ labelIndex, setLabels, removeRow, errors }): ReactElement => {
+  const onLabelChange = (event: ChangeEvent<HTMLInputElement>): void => {
+    setLabels(labelIndex, event.target.value);
+  };
+
+  return (
+    <div className={classes.deleteRowContainer}>
+      {/* eslint-disable-next-line */}
+      <div title="Удалить эту строку" className={classes.closeX} onClick={() => removeRow(labelIndex)} />
```

Comment:
Adding click event to div is not good idea, it is bad for accessibility, it should be a button or a link instead

- [ ]

Reply:
Done!